snap/snapcraft.yaml

```diff
@@ -0,0 +1,35 @@
+name: lazygit
+title: Lazygit
+base: core24
+adopt-info: lazygit
+summary: simple terminal UI for git commands
+description: A simple terminal UI for git commands, written in Go with the gocui library.
+version: '0.42.0'
```

Comment:
Is this something I should be bumping whenever I do a release or is this depending on whoever manages the snap entity?

Reply:
It's completely up to you, you can simply set the latest release tag number as the version, or you can follow something completely different.

Reply:
generally, what I do is, I set the version as the release numbers.